static/app/views/replays/replayTable/tableCell.tsx

```diff
@@ -374,21 +374,25 @@ export function ReplayCell({
 
   return (
     <Item isWidget={isWidget} isReplayCell className={className}>
-      <UserBadge
-        avatarSize={24}
-        displayName={
-          replay.is_archived ? (
-            replay.user.display_name || t('Anonymous User')
-          ) : (
-            <MainLink to={detailsTab} onClick={trackNavigationEvent}>
-              {replay.user.display_name || t('Anonymous User')}
-            </MainLink>
-          )
-        }
-        user={getUserBadgeUser(replay)}
-        // this is the subheading for the avatar, so displayEmail in this case is a misnomer
-        displayEmail={subText}
-      />
+      <Row gap={1}>
+        <UserAvatar user={getUserBadgeUser(replay)} size={24} />
+        <SubText>
```

Comment:
Does this look different than using `UserBadge`?

Reply:
it's looking the same to me:

<img width="349" alt="SCR-20240411-ivmn" src="https://github.com/getsentry/sentry/assets/56095982/7eb9b17c-6805-4e78-80e0-0dcb6fa04e41">

Reply:
i'll followup soon with some stories to differentiate the avatar stuff. There's too many components related to that :(